public/admin/config.yml

```diff
@@ -47,7 +47,40 @@ collections:
                   - { label: 'Alt text', name: 'altText', widget: 'string' }
                   - { label: 'Image', name: 'image', widget: 'image' }
                   - { label: 'Url', name: 'url', widget: 'string' }
-
+          - {
+              label: 'Who we serve: Header',
+              name: 'whoWeServeHeader',
+              widget: 'string',
+            }
+          - {
+              label: 'Who we serve: Content',
+              name: 'whoWeServeContent',
+              widget: 'markdown',
+            }
```

Comment:
Consider adding a `sectionID` to each section on this page (we might need to add a table of contents/sidebar here).

Reply:
good call

Reply:
update: punting this for now